.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: "Version to use for the release."
+        required: true
+        default: "X.Y.Z"
+      developmentVersion:
+        description: "Version to use for further development"
+        required: true
+        default: "X.Y.Z-RIE"
+jobs:
+  Release:
+    environment: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
Did you check if this checks out `main` branch or `develop` ? We always want to release from `main`

Reply:
Good catch, thank you!

Reply:
So I made it so when you run the workflow, there's a dropdown to select what branch you want to release from and even if you select "develop" branch instead of "main" it will checkout the "main" branch. Is that ok?

Reply:
I sounds like it could be confusing for someone who's not familiar with that, but we still really want to release from main, so it's fine.